app/lib/tcomb-form/types/localImage.js

```diff
@@ -0,0 +1,16 @@
+import t from 'tcomb-form-native';
+
+const LocalImage = t.struct({
```

Comment:
Custom types for the form are uppercased in tcomb read me. I think it is some convention to separate type from instance.